projects/plugins/jetpack/to-test.md

```diff
@@ -1,45 +1,201 @@
+### Earn
 
-### Newsletters
+There's a new Earn section added to the Jetpack Settings dashboard. It should contain the Ads section previously located in Settings > Traffic. Depending on whether a user is connected or not it should offer to connect before showing payments related information. Try loading this page with a disconnected site, connecting, and reviewing its contents. Make sure all links work, and there are no visual problems.
 
-The Newsletter functionality has received various upgrades including a new email preview feature. To test:
+### Subscribers
 
-- In `Jetpack > Settings > Discussion`, make sure that Subscriptions are enabled.
-- Create a new test post and click on Publish. In the publishing panel you will see the "Newsletter visibility" section which has been updated to clarify the audience reach of your post.
-- Try clicking on the Preview button in that section to send a test email preview to the current user.
+We currently link to an old Calypso page when folks want to see their subscribers after enabling the Subscriptions. But in the past few months, there were two new screens added that can be better than the existing one:
 
-### Tock Block
+- Your site will need to be connected to WordPress.com, and the Subscriptions module must be enabled.
+- Go to Jetpack > Settings > Discussion
+- Ensure the link to view subscribers works. It should be updated depending on your site settings
+  - Is the Stats module enabled (you can disable it under Jetpack > Settings > Module)
+  - Is the new Stats experience enabled (you can check that under Jetpack > Settings > Traffic
 
-A new block is now available for the Tock reservations service:
+### Custom CSS
 
-- While editing a post, search for and add the new Tock block.
-- The block will ask you to specify the business name, for testing purposes you can try `oquirrh`
-- Once your post is published, clicking on the Tock button on the frontend of your site will open a modal for booking.
+There's a new feature introduced into Jetpack to detect if the WordPress site has the core site editor feature enabled. With it comess a contextual adjustment to the user interface. The Jetpack > Settings > Writing > Custom CSS toggle is now conditionally rendered based on whether the site uses a block-based theme or not. For sites using a block-based theme, this toggle is removed, aligning with the concept that such themes inherently offer greater flexibility for customization and thus the additional CSS settings might not be necessary.
 
-### Sharing Buttons
+This PR involves changes in multiple files, including state reducers, the writing settings component, and theme enhancements, to ensure the changes are effectively integrated.
 
-We've added a new sharing button for the Nextdoor service:
+This is classified as a minor enhancement due to its impact on improving the user interface and experience when interacting with block-based themes.
```

Comment:
Do you need those extra explanations here?

Reply:
You're right, it's not necessary, I'm going to remove them.